packages/main/src/RangeSlider.js

```diff
@@ -80,6 +80,22 @@ const metadata = {
  * <h3>Responsive Behavior</h3>
  * You can move the currently selected range by clicking on it and dragging it along the interval.
  *
+ * <h3>Keyboard Handling</h3>
+ *
+ * <ul>
+ * <li><code>Left or Down Arrow</code> - Moves a <code>ui5-range-slider</code> handle or the entire selection one step to the left;</li>
```

Comment:
Instead of a tag name, use ```the component``` everywhere in the docs. This way there will be no issues with the docs of inherited controls.

Reply:
Done